www/audit.py

```diff
@@ -703,7 +720,7 @@ def api_feature(pid):
     project = Project.get(Project.id == pid)
     if user and request.method == 'POST' and project.can_validate:
         ref_and_audit = request.get_json()
-        if ref_and_audit and len(ref_and_audit) == 2:
+        if ref_and_audit and len(ref_and_audit) == 3:
```

Comment:
Wenn wir das schon anfassen: Magic number benennen: für was steht diese 3?

Reply:
`3` steht hier für die Anzahl 3, denn der verglichene Wert stammt aus einem `len(…)`-Aufruf. Von dem her ist es keine "Magic Number".

Interessant wäre allerdings, _welche_ 3 Sachen man in der Rückgabe von `request.get_json()` erwartet, was m.M.n. am besten durch ein destructuring assignment (siehe auch https://github.com/geometalab/cf_audit/pull/2#discussion_r1040950903) kommuniziert würde.

Reply:
Für mich ist es aus deinen genannten Gründen genu eine magische Zahl, da es nicht es nicht ersichtlich ist, weshalb es genau drei braucht. Wenn das oben angehen, ist das auch gelöst, denn dann müsste man diesen Vergleich nicht mehr machen. ;-)